recipes/dcase2021_task4_baseline/confs/sep+sed.yaml

```diff
@@ -0,0 +1,86 @@
+training:
+  #batch size: [synth, weak, unlabel]
+  batch_size: [1, 2, 2]
+  batch_size_val: 1
+  const_max: 2 # max weight used for self supervised loss
+  n_epochs_warmup: 30 # num epochs used for exponential warmup
+  num_workers: 6 # change according to your cpu
+  n_epochs: 200 # max num epochs
+  early_stop_patience: 5 # Same as number of epochs by default, so no early stopping used
+  accumulate_batches: 64
+  gradient_clip: 5. # 0 no gradient clipping
+  median_window: 7 # length of median filter used to smooth prediction in inference (nb of output frames)
+  val_thresholds: [0.5] # thresholds used to compute f1 intersection in validation.
+  n_test_thresholds: 50 # number of thresholds used to compute psds in test
+  ema_factor: 0.999 # ema factor for mean teacher
+  self_sup_loss: mse # bce or mse for self supervised mean teacher loss
+  backend: # pytorch lightning backend, ddp, dp or None
+  validation_interval: 5 # perform validation every X epoch, 1 default
+  weak_split: 0.9
+  seed: 42
+  mixup: # Soft mixup gives the ratio of the mix to the labels, hard mixup gives a 1 to every label present.
+  obj_metric_synth_type: intersection
+  sed_yaml: "./pretrained_SED/version_0/hparams.yaml"
```

Comment:
Probably we can clarify in the README that they need to download `2021_baseline_sed.tar.gz`, and not just the checkpoint (otherwise they won't have this, and probably to check the paths).